rel/overlay/etc/default.ini

```diff
@@ -930,3 +930,97 @@ url = {{nouveau_url}}
 ;max_objects = 10000
 ;max_idle = 600000
 ;enable = true
+
+[couch_scanner]
+; How often to check for configuration changes and start/stop plugins
+;interval_sec = 5
+
+; Minimum time to force a plugin to wait before running again after a crash
+;min_penalty_sec = 30
+
+; Maximum time to force a plugin to wait after repeated crashes (8 hours default)
+;max_penalty_sec = 28800
+
+; If plugin runs successfully without crashing for this long, reset its
+; repeated error count
+;heal_threshold_sec = 300
+
+; Database processing rate limit. This will also be the rate at which design
+; documents are fetched. The rate is shared across all running plugins.
+;db_rate_limit = 50
+
+; Limits the rate at which plugins may open db shard files on a node. The
+; rate is shared across all running plugins.
+;shard_rate_limit = 50
```

Comment:
If a plugin is only interested in database names, and doesn't actually open any dbs or docs, does this rate then not apply? Would such a plugin go as fast as possible in that case?

Reply:
If a plugin is only interested in database names it wouldn't open shard at all, it might return `[]` for `shards(...)` callback to indicate that. Then only the db rate would apply. If a plugin uses the shards and just looks at their sizes or doc ids then indicates to `skip` opening the doc, then only db rate and shard open rate would apply but not the doc rate.